test/Libraries/DynamoPythonTests/CodeCompletionTests.cs

```diff
@@ -434,5 +429,19 @@ public void VerifyIronPythonLoadedAssemblies()
                 }
             }
         }
+
+        private void verifyImportedTypes(Dictionary<string, Type> importedTypes)
+        {
+            var defaultImports = new string[]
+            {
+                "System",
+                "None"
+            };
+
+            foreach (var type in importedTypes)
```

Comment:
this seems to reimplement sequence equals:
https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.sequenceequal?view=netframework-4.7.2 ?

any reason not to just use it?

Reply:
Should be good now